modules/vstudio/tests/vc2010/test_compile_settings.lua

```diff
@@ -428,6 +428,34 @@
 		]]
 	end
 
+--
+-- If ConformanceMode flag is set, add <ConformanceMode> element (supported from VS2017)
+--
+
+	function suite.onConformanceMode_VS2015()
```

Comment:
This test doesn't confirm that `<ConformanceMode>` isn't output, you're just checking that everything up to the conformance mode element is output. You need to check that the element that would come after it is there, whatever that happens to be.

Reply:
Could you suggest a correct way to check for absense of a flag? Relying on a default project properties is not reliable probably.

Reply:
Like I said, you can just check for the element that is output after it. Otherwise, you can use `test.excludes("<ConformanceMode>true</ConformanceMode>", p.captured())`.

Reply:
`test.excludes()` is better here I think. Thank you.